kafka_consumer/datadog_checks/kafka_consumer/client/confluent_kafka_client.py

```diff
@@ -35,7 +35,10 @@ def reset_offsets(self):
         raise NotImplementedError
 
     def get_partitions_for_topic(self, topic):
-        raise NotImplementedError
+        cluster_metadata = self.kafka_client.list_topics(topic)
+        topics = cluster_metadata.topics
+        partitions = list(topics[topic].partitions.keys())
+        return partitions or []
```

Comment:
This will fail if `topic` is not found because of `topics[topic]` right?

Reply:
Updated to match `kafka_python` implementation!